Makefile

```diff
@@ -85,12 +85,12 @@ benchmarks:
 .PHONY: bench-old
 bench-old: benchmarks
 	@echo "Benchmarking old engine"
-	@go test ./... -bench 'BenchmarkRangeQuery/.*/old_engine'  -run none -count 10 | sed -u 's/\/old_engine//' > benchmarks/old.out
+	@go test ./... -bench 'BenchmarkRangeQuery/.*/old_engine'  -run none -count 5 | sed -u 's/\/old_engine//' > benchmarks/old.out
```

Comment:
Now that the amount of benchmarks has grown, we can reduce this number to get results faster. Five iterations seem to be enough for statistical significance.